modules/installation-creating-aws-subnet-localzone.adoc

```diff
@@ -35,47 +35,29 @@ requires:
 ----
 [
   {
-    "ParameterKey": "ClusterName", <1>
-    "ParameterValue": "mycluster" <2>
+    "ParameterKey": "VpcId",
+    "ParameterValue": "<value_of_VpcId>" <1>
   },
   {
-    "ParameterKey": "VpcId", <3>
-    "ParameterValue": "vpc-<random_string>" <4>
+    "ParameterKey": "PublicRouteTableId",
+    "ParameterValue": "<value_of_PublicRouteTableId>" <2>
   },
   {
-    "ParameterKey": "PublicRouteTableId", <5>
-    "ParameterValue": "<vpc_rtb_pub>" <6>
+    "ParameterKey": "LocalZoneName",
+    "ParameterValue": "<value_of_GroupName>" <3>
   },
   {
-    "ParameterKey": "LocalZoneName", <7>
-    "ParameterValue": "<cluster_region_name>-<location_identifier>-<zone_identifier>" <8>
-  },
-  {
-    "ParameterKey": "LocalZoneNameShort", <9>
-    "ParameterValue": "<lz_zone_shortname>" <10>
-  },
-  {
-    "ParameterKey": "PublicSubnetCidr", <11>
-    "ParameterValue": "10.0.128.0/20" <12>
+    "ParameterKey": "PublicSubnetCidr",
+    "ParameterValue": "10.0.192.0/20" <4>
   }
```

Comment:
According to the template, `SubnetName` ParameterKey and ParameterValue are missing.

Reply:
@yunjiang29 Could you GitHub suggest the change that would fix this? Most of this content predates my involvement. 

Is it just:

```
"ParameterKey": "SubnetName",
"ParameterValue": "<value>"
```

Where `<value>` is something that the user should know and enter, like `cluster-public-us-east-1-nyc-1a`, or is more guidance available?

Reply:
> Where <value> is something that the user should know and enter, like cluster-public-us-east-1-nyc-1a

@maxwelldb Yes, it's the name of Local Zone Subnet, like `cluster-public-us-east-1-nyc-1a`

> @yunjiang29 Could you GitHub suggest the change that would fix this? Most of this content predates my involvement.

@maxwelldb sorry for inconvenience, please see https://github.com/openshift/openshift-docs/pull/57427/files#r1188046537

Reply:
Thanks!